jobmanager/Job.h

```diff
@@ -6,13 +6,30 @@
 namespace jobmanager
 {
 
-class Job
+class JobBase
+{
+public:
+    JobBase() = default;
+    virtual ~JobBase() {}
+
+    // return true if job needs to be run again
+    virtual bool runOnce() = 0;
+};
+
+class Job : public JobBase
 {
```

Comment:
These two classes whould swap names, but I did not want to cause all the refactoring atm.